docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -192,6 +193,37 @@ test(`Test Speed`, async t => {
 });
 ```
 
+### Setting Page Load Timeout
+
+Page load timeout defines the amount of time passed after the `DOMContentLoaded` event, within which the `window.load` event should be raised.
+
+After the timeout passes or the `window.load` event is raised (whatever happens first), TestCafe starts the test.
+
+To specify the page load timeout in test code, use the `t.setPageLoadTimeout` method.
+
+```text
+t.setPageLoadTimeout( duration )
+```
+
+Parameter  | Type      | Description
+---------- | --------- | -----------
+`duration` | Number    | Page load timeout, in milliseconds. `0` to skip waiting for the `window.load` event.
+
+You can also set the page load timeout when launching tests via the [command line](../using-testcafe/command-line-interface.md#--page-load-timeout-ms) or [API](../using-testcafe/programming-interface/runner.md#run).
+
+**Example**
+
+```js
+fixture `Test Speed`
```

Comment:
Specify appropriate names for fixture and test, and it's not a 'Test Speed' test, it's 'Page Load Timeout'.